knowledge-base/fiddler-as-reverse-proxy.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Using Fiddler Everywhere as a Reverse Proxy
+description: "Use Fiddler Everywhere as a local reverse proxy for testing HTTP/1 server."
+type: how-to
+slug: fiddler-reverse-proxy
+publish: true
+res_type: kb
+---
+
+
+One common scenario for developers, QA, and support engineers is to test how their server behaves when a client request is received. A common technique is the usage of reverse proxies, where the proxy tool is installed on the server, and all inbound HTTP requests explicitly go through it before landing on the server. Using a standalone reverse proxy can be hard to set up and a burden to configure for cases when you like only to test and inspect the received HTTP requests. Additionally, in the usual case, developers use localhost servers to mock and test their web applications, so setting up a full-blown reverse proxy can be over complicated. 
```

Comment:
`cases when you like only to test and inspect the received HTTP requests` - the word `like` could be replaced with `want` or `would like`.